xml/gpu_passthru.xml

```diff
@@ -0,0 +1,503 @@
+      card and storage.
+     </para>
+    </listitem>
+   </itemizedlist>
+  </sect2>
+
+  <sect2 xml:id="gpu-passthru-guest-driver">
+   <title>Install the graphic card driver</title>
+   <sect3 xml:id="gpu-passthru-guest-driver-linux-rpm">
+    <title>Linux guest</title>
+    <procedure>
+     <title>RPM-based distributions</title>
+     <step>
+      <para>
+       Download the driver RPM package from
+       <link xlink:href="http://www.nvidia.com/download/driverResults.aspx/131159/en-us"/>.
+      </para>
+     </step>
+     <step>
+      <para>
+       Install the downloaded RPM package:
+      </para>
+<screen>&prompt.sudo;rpm -i nvidia-diag-driver-local-repo-sles123-390.30-1.0-1.x86_64.rpm</screen>
+     </step>
+     <step>
+      <para>
+       Refresh repositories and install <package>cuda-drivers</package>. This
+       step will be different for non-&suse; distributions:
+      </para>
+<screen>&prompt.sudo;zypper refresh &amp;&amp; zypper install cuda-drivers</screen>
+     </step>
+     <step>
+      <para>
+       Reboot the guest VM:
+      </para>
+<screen>&prompt.sudo;shutdown -r now</screen>
+     </step>
+    </procedure>
+    <procedure>
+     <title>Generic installer</title>
+     <step>
+      <para>
+       Because the installer needs to compile the &nvidia; driver modules,
+       install <package>gcc-c++</package> and <package>kernel-devel</package>
+       packages.
+      </para>
+     </step>
+     <step>
+      <para>
+       Disable Secure Boot on the guest, because the &nvidia; driver modules are not
+       signed. On &suse; distributions, you can use &yast; &grub; module to
+       disable Secure Boot. Find more information in
+       <xref linkend="sec-uefi-secboot-sle"/>.
+      </para>
+     </step>
+     <step>
+      <para>
+       Download the driver installation script from
+       <link xlink:href="https://www.nvidia.com/Download/index.aspx?lang=en-us"/>,
+       make it executable, and execute it, and complete the driver
+       installation:
+      </para>
+<screen>
+&prompt.user;chmod +x NVIDIA-Linux-x86_64-460.73.01.run
+&prompt.sudo;./NVIDIA-Linux-x86_64-460.73.01.run
+</screen>
+     </step>
+     <step>
+      <para>
+       Download CUDA drivers from
+       <link xlink:href="https://developer.nvidia.com/cuda-downloads?target_os=Linux&amp;target_arch=x86_64&amp;target_distro=SLES&amp;target_version=15&amp;target_type=rpmlocal"/>
+       and install following the on-screen instructions.
+      </para>
+     </step>
+    </procedure>
+    <note>
+     <title>Display issues</title>
+     <para>
+      After you have installed the &nvidia; drivers, the &vmm; display will
+      lose connection. To access the guest VM, you need to login via
+      <command>ssh</command>, change to console interface, or install a
+      dedicated VNC server in the guest. To avoid a flickering screen, stop and
+      disable the display manager:
```

Comment:
```suggestion
      After you have installed the &nvidia; drivers, the &vmm; display will
      lose its connection to the guest OS. To access the guest VM, you
      must either login via <command>ssh</command>, change to the
      console interface, or install a dedicated VNC server in the guest.
      To avoid a flickering screen, stop and disable the display manager:
```